packages/react/tests/e2e-storybook/cypress/integration/CTA/Feature.e2e.js

```diff
@@ -172,7 +172,7 @@ describe('dds-feature-cta | (desktop)', () => {
   });
 
   it('Should load and be fully clickable', _tests.checkBlockLink);
-  it(
+  xit(
```

Comment:
should we add a note about this omitted test?

Reply:
Ahh just saw your note after the PR was merged. I'll open up a PR with a note.

Reply:
Actually let me create an issue for this too.